### PR TITLE
NF-85 - wire up posting create claim to stub

### PR DESCRIPTION
**Pull request by SteveSugden**
- using EIS models created by NDRC as starting point

### CODE REVIEW

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentre/connectors/CreateCaseConnector.scala [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  def submitClaim(request: EISCreateCaseRequest, correlationId: String)(implicit
+    hc: HeaderCarrier,
+    ec: ExecutionContext
+  ): Future[EISCreateCaseResponse] =
+    http.POST[EISCreateCaseRequest, EISCreateCaseResponse](url, request)(
+      implicitly[Writes[EISCreateCaseRequest]],
+      readFromJsonSuccessOrFailure,
+      HeaderCarrier(authorization = Some(Authorization(s"Bearer ${config.eisAuthorizationToken}")))
+        .withExtraHeaders(
```
Comment (crosslandwa) [inline app/uk/gov/hmrc/nationalimportdutyadjustmentcentre/connectors/CreateCaseConnector.scala:50]: OBSERVATION: These headers differ to what we have in the API spec we were sent from the downstream service....

However, i'm happy to approve/leave them as they are for now (and we can adjust them later when we know more)

Comment (SteveSugden) [reply app/uk/gov/hmrc/nationalimportdutyadjustmentcentre/connectors/CreateCaseConnector.scala:50]: Maybe they are the EIS spec headers (for NDRC)?

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentre/connectors/PegaConnector.scala [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+/** Provides PEGA API headers */
+trait PegaConnector {
+
+  final val httpDateFormat = DateTimeFormatter
+    .ofPattern("EEE, dd MMM yyyy HH:mm:ss z", ju.Locale.ENGLISH)
+    .withZone(ZoneId.of("GMT"))
+
+  /** Headers required by the PEGA API */
+  final def pegaApiHeaders(correlationId: String, environment: String): Seq[(String, String)] =
```
Comment (crosslandwa) [inline app/uk/gov/hmrc/nationalimportdutyadjustmentcentre/connectors/PegaConnector.scala:31]: OBSERVATION: you can use this in the `withExtraHeaders` call in the `CreateCaseConnector` (or delete it, given its unused)

Comment (SteveSugden) [reply app/uk/gov/hmrc/nationalimportdutyadjustmentcentre/connectors/PegaConnector.scala:31]: Excellent spot.    I guess I'd better use that method seeing how I copied it.

**Review by crosslandwa — DISMISSED** [review]

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentre/connectors/PegaConnector.scala (hunk 2 above)
Comment (SteveSugden) [reply app/uk/gov/hmrc/nationalimportdutyadjustmentcentre/connectors/PegaConnector.scala:31]: I addressed this just about the same time as you approved.  Just not quite fast enough.

**Review by crosslandwa — APPROVED** [review]

**Pull request merged**